repos.md

```diff
@@ -5,10 +5,18 @@
 Trailmix. A private place to write. [https://www.trailmix.life](https://www.trailmix.life)  
 [https://github.com/codecation/trailmix](https://github.com/codecation/trailmix)
 
+### lobsters
+Computing-focused community centered around link aggregation and discussion [https://lobste.rs](https://lobste.rs)  
+[https://github.com/jcs/lobsters](https://github.com/jcs/lobsters)
```

Comment:
@katelovescode I believe https://github.com/lobsters/lobsters is more up to date. 

```suggestion
[https://github.com/jcs/lobsters](https://github.com/lobsters/lobsters)
```